src/route/dataset-route.ts

```diff
@@ -131,6 +148,11 @@ apiRoute.get('/:dataset/csv', async (req, res) => {
         res.json({ message: 'Dataset not found... Dataset ID not found in Database' });
         return;
     }
+    if (isValidUUID(datasetID) === false) {
```

Comment:
Copypasta? Why do we do this check again?

Reply:
Ah... it's another route handler.
Perhaps factor this stuff out as it appears in another handler again below.

Reply:
Moved to a new function